cqproto/internal/plugin.proto

```diff
@@ -24,8 +24,6 @@ message ConfigureProvider {
     ConnectionDetails connection = 2;
     // Holds information such as credentials, regions, accounts, etc'
     bytes config = 3;
-    // Msgpack encoded: Allows to inject & override fields into resource tables, use this carefully to override fields. Used in tests.
-    bytes extraFields = 5;
```

Comment:
not 100% sure, but wouldn't this break prior versions? wouldn't it be better to do [deprecated = true] instead, and when decoding / encoding with new SDK ignore it?

Reply:
it would just show up as default empty value which would be a nil slice and map to a nil map.